libs/s25main/mapGenerator/Resources.cpp

```diff
@@ -177,39 +177,65 @@ namespace rttr { namespace mapGenerator {
     {
         const auto& textures = map.textureMap;
         auto& resources = map.resources;
+        struct mineableResourceInfo
```

Comment:
Type names should be UpperCamelCase
```suggestion
        struct MineableResourceInfo
```